src/BlockchainReader/Bitcoin.test.ts

```diff
@@ -0,0 +1,110 @@
+import { PoetTimestamp } from '@po.et/poet-js'
+import { allPass, equals } from 'ramda'
+import { describe } from 'riteway'
+
+import { PREFIX_POET, PREFIX_BARD } from 'Helpers/Bitcoin'
+
+import { blockToPoetAnchors } from './Bitcoin'
+
+import * as TestBlock from './TestData/block-00000000000151360aad32397ff1cf7dd303bed163b0ef425e71a53ccdec7312.json'
+
+describe('Bitcoin.blockToPoetAnchors', async should => {
+  const { assert } = should()
+
+  assert({
+    given: 'testnet block 00000000000151360aad32397ff1cf7dd303bed163b0ef425e71a53ccdec7312',
```

Comment:
Little less specific here and a bit mored descriptive would be a sligh improvement I think, `given: a 'valid testnet block'`.

Reply:
I'm actually validating things that are only true for this particular block, though.

Do you still feel it's better to be less specific? (I'll change it if you do)

Reply:
Maybe if I change the hash for the height? (Less aggressive to the eye?)

Reply:
See [this commit](https://github.com/poetapp/node/pull/383/commits/242156343ab0031783ff369a517e1750144bf869)

Reply:
Yup that helps make it less aggressive. Although, I would like given to explain what is special about this particular block if possible, `given a block that ...`, so that reading the test I know what is special about it.